src/main/drivers/pwm_output_dshot.c

```diff
@@ -174,12 +187,17 @@ void pwmCompleteDshotMotorUpdate(void)
     }
 }
 
+#if defined(STM32F3)
+CCM_CODE
+#else
+FAST_CODE
+#endif
 static void motor_DMA_IRQHandler(dmaChannelDescriptor_t *descriptor)
 {
     if (DMA_GET_FLAG_STATUS(descriptor, DMA_IT_TCIF)) {
         motorDmaOutput_t * const motor = &dmaMotors[descriptor->userParam];
 #ifdef USE_DSHOT_TELEMETRY
-        uint32_t irqStart = micros();
+        dshotDMAHandlerCycleCounters.irqAt = DWT->CYCCNT;
```

Comment:
`DWT->CYCCNT` should be probably encapsulated in some (inline) function

Reply:
this comment should be on #8768

Reply:
also, yes I will do that.

Reply:
done